src/plugins/recordTypes/movement/forms/default.jsx

```diff
@@ -0,0 +1,55 @@
+const template = (configContext) => {
+  const {
+    React,
+  } = configContext.lib;
+
+  const {
+    Col,
+    Cols,
+    Panel,
+    Row,
+  } = configContext.layoutComponents;
+
+  const {
+    Field,
+  } = configContext.recordComponents;
+
+  return (
+    <Field name="document">
+      <Panel name="location" collapsible>
```

Comment:
This looks weird. The first row doesn't line up with anything. Let's just do two columns:

```
locationDate     |  movementNote
reasonForMove    |
currentLocation  |
movementContact  |
```